pkgs/by-name/mo/monophony/package.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, fetchFromGitLab
+, python3Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
+, libadwaita
+, libsoup_3
+, glib-networking
+,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "monophony";
+  version = "2.3.0";
+  format = "other";
+
+  sourceRoot = "source/source";
+  src = fetchFromGitLab {
+    owner = "zehkira";
+    repo = "monophony";
+    rev = "v${version}";
+    hash = "sha256-vAX9ac706fKVYW1jjgUJ2GvGRWeytYfgf7Lal+gEm04=";
```

Comment:
```suggestion
    hash = "sha256-n9tioIXMjUs9AmNuVLrOHJqeGhotkFlv+bTeHlsQPaM=";
```
hash needs to be updated for new version.

Reply:
Oh huh

Reply:
How did it build?

Reply:
Without me doing that how did it still build and run

Reply:
Fixed

Reply:
it's an FOD, you must always blank the hash manually when updating version

Reply:
Oh okay. What is an FOD?

Reply:
Thanks!